File: remoteprocess/src/main/java/com/anmnight/remoteprocess/MainActivity.java
package com.anmnight.remoteprocess;

import android.app.Activity;
import android.os.Bundle;

import com.anmnight.remoteprocess.unit.RoomLoginUnit;
import com.anmnight.remoteprocess.works.SystemLogWorker;

import java.util.concurrent.TimeUnit;

import androidx.work.PeriodicWorkRequest;
import androidx.work.WorkManager;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        PeriodicWorkRequest mPeriodicWorkRequest = new PeriodicWorkRequest.Builder(SystemLogWorker.class, 15, TimeUnit.MINUTES)
                .build();


        WorkManager.getInstance().enqueue(mPeriodicWorkRequest);

        final RoomLoginUnit loginUnit = new RoomLoginUnit();

        loginUnit.login(this);


    }
}

File: host/src/main/java/com/example/testapp/activities/MainActivity.kt
package com.example.testapp.activities

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.example.testapp.R
import kotlinx.android.synthetic.main.activity_test_main.*

class MainActivity : AppCompatActivity() {

    private val MAIN_ENTERY_NAME = "Entry"
    private val tag = "MainActivity"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_test_main)


        text.scrollTo(0, 100)

        Log.i(tag, "im child");
    }

}
